src/os/file/file.js

```diff
@@ -440,3 +444,14 @@ os.file.deserializeFile = function(data) {
 os.file.serializeFile = function(file) {
   return file ? file.persist() : undefined;
 };
+
+
+/**
+ * Normalizes all whitespace excluding newlines to std space
+ * @param {?(ArrayBuffer|string)} string
+ * @return {string}
+ */
+os.file.normalizeWhitespace = function(string) {
+  // Convert whitespace (-newlines) to standard space
+  return string.replace(/[^\S\n]/g, '\u0020');
```

Comment:
I do not think this is a good idea. Some files are tab-delimited. Let's discuss what you were trying to fix.